integration/template/kvm_operator_charts_values.go

```diff
@@ -5,6 +5,9 @@ package template
 // variables will be expanded before writing the contents to a file.
 var KVMOperatorChartValues = `Installation:
   V1:
+    GiantSwarm:
+      KVMOperator:
+        LabelSelector: 'giantswarm.io/cluster=${CLUSTER_NAME}'
```

Comment:
Aligns with the installation values. Note that I also changed the label selector according to the new cluster ID label. Before we used the deprecated one. Should be good due to https://github.com/giantswarm/certs/blob/master/k8s.go#L70-L77.